mods/mc_helpers/init.lua

```diff
@@ -123,3 +123,28 @@ function mc_helpers.pairsByKeys (t, f)
     end
     return iter
 end
+
```

Comment:
MineTest provides some [helper functions](https://minetest.gitlab.io/minetest/helper-functions) which seem to be very similar in functionality to the newly-added helper functions:

- `mc_helpers.trim(s)` and `string:trim()`
- `mc_helpers.deepCopy(table)` and `table.copy(table)`

That being said, it may still be good to keep these implementations so that we have helper functions which have more predictable behaviours

Reply:
These versions have been implemented because the minetest versions are not JIT compiled in lua JIT versions of Minetest, increasing context switching and reducing performance.

Reply:
Noted; will make a note to refactor calls to `string:trim()` and `table.copy()` in some other mods once this branch has been merged